lib/cli/watch.js

```diff
@@ -17,9 +18,16 @@ export default async function watch(options = {}) {
 
   const server = new Server(reptar);
   await server.start();
-
   activity.end(startActivity);
 
   process.stdout.write('\n');
   log.info('Server running at:', server.server.info.uri);
+
+  if (reptar.options.browserSync) {
```

Comment:
I changed this to get the options from the watch function. That makes perfect sense. 

I'm not sure I follow about disabling browserSync. I uninstalled it globally on my machine, which did cause a problem. But I just needed to `npm install` in my test Reptar project and it worked like a charm.